spec/furniture/marketplace/stripe_events_controller_request_spec.rb

```diff
@@ -81,6 +97,22 @@
         expect(Stripe::Transfer).not_to(have_received(:create))
       end
     end
+
+    context "when Square notifications are not enabled" do
+      it "does not attempt to transfer the order to seller's Square dashboard" do
```

Comment:
FYI: Unless we record real http interactions with Square (do we want to now, ever, using `vcr` or something else?) I like using the term "attempt" to more accurately describe what the test covers.

Reply:
I don't find usages of `attempt` or similar particularly useful. The test describes what it does. Whether or not it is mocked feels incidental.

Reply:
I don't have a strong preference either way, though I like the embellishment because it reads more like natural language for tossing data into a void. Though I'm privvy to a more literary approach to any documentary writing. I can remove it at least so it's consistent with other writing.